### PR TITLE
951 - Dashboard Task Row Clickable

### PR DESCRIPTION
**Pull request by sarvaiyanidhi**
# 🔗 Issue
#951 

# ✍️ Description
- Code changes to make entire row clickable for result displayed for overdue and incomplete tasks

# 📷 Screenshots/Demos
NA

### CODE REVIEW

File: app/views/organizations/staff/dashboard/_pets_with_incomplete_or_overdue_tasks.html.erb [hunk 1]
```diff
@@ -11,13 +11,13 @@
       </thead>
       <tbody>
         <% @pets.each do |pet| %>
-          <tr>
+          <tr data-controller="click-row" data-click-row-url-value="<%= staff_pet_path(pet, active_tab: 'tasks') %>" class="cursor-pointer">
```
Comment (kasugaijin) [inline app/views/organizations/staff/dashboard/_pets_with_incomplete_or_overdue_tasks.html.erb:14]: Do we need to use JavaScript for this? Can't we just wrap the row in a `link_to` helper that has params for the `pet.id` and the `active_tab`?

Comment (sarvaiyanidhi) [reply app/views/organizations/staff/dashboard/_pets_with_incomplete_or_overdue_tasks.html.erb:14]: I don't think we can wrap link_to directly with tr tag so went for this approach. I still checked again and implemented it but it does not work. Let me know if I am missing something and if there is simple solution to this.

Comment (kasugaijin) [reply app/views/organizations/staff/dashboard/_pets_with_incomplete_or_overdue_tasks.html.erb:14]: @sarvaiyanidhi ah I did not know that! Today I learned...
In that case, can we wrap the pet name, sex and count in the same link_to tag? I think it's a much simpler solution than adding a new JS controller.

Comment (sarvaiyanidhi) [reply app/views/organizations/staff/dashboard/_pets_with_incomplete_or_overdue_tasks.html.erb:14]: Sure, adding link_to to each value of column will be straight forward.. Will make changes and push code.

**Review by kasugaijin — APPROVED** [review]
Thank you @sarvaiyanidhi

**Pull request merged**